sdktests/server_side_persistence_base.go

```diff
@@ -572,72 +572,10 @@ func (s *ServerSidePersistentTests) Run(t *ldtest.T) {
 
 					require.NoError(t, s.persistentStore.WriteMap(s.defaultPrefix, "features", s.initialFlags))
 
-					if cacheConfig.Mode == servicedef.CacheModeInfinite {
-						// This key was already cached, so it shouldn't see the change above.
-						h.RequireNever(t,
-							checkForUpdatedValue(t, client, "flag-key", context,
-								ldvalue.String("value"), ldvalue.String("new-value"), ldvalue.String("default")),
-							time.Millisecond*500, time.Millisecond*20, "flag-key was incorrectly updated")
```

Comment:
The diff looks weird, but basically the first two if / else if blocks are reduced to this single check.

In FDv2, if the data system has a data source that has provided data, we never rely on the  persistent store again. As a result, the "But since ..." checks weren't valid.